src/wagtailtrans/templates/modeladmin/wagtailtrans/language/delete.html

```diff
@@ -0,0 +1,41 @@
+{% extends "wagtailadmin/base.html" %}
```

Comment:
Please extend from the existing ``delete.html`` template. This way you only have to override ``content_main`` and we keep the most of wagtails interface intact.